release.json

```diff
@@ -7,6 +7,22 @@
 		"JMXFETCH_HASH": "078011fbcca5e9beee2f041fefacbad824cff47e074b74a4674eb3efed8984b3",
 		"PYTHON_RUNTIMES": "3"
 	},
+	"7.15.0-rc.1": {
+		"INTEGRATIONS_CORE_VERSION": "6.15.0-rc.1",
+		"OMNIBUS_SOFTWARE_VERSION": "6.15.0-rc.1",
+		"OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+		"JMXFETCH_VERSION": "0.33.0",
+		"JMXFETCH_HASH": "078011fbcca5e9beee2f041fefacbad824cff47e074b74a4674eb3efed8984b3",
+		"PYTHON_RUNTIMES": "3"
```

Comment:
I believe this is not needed anymore, we set the `PYTHON_RUNTIMES` variable in the jobs themselves now in the `.gitlab-ci.yml` now. But I also think leaving it here is fine, so no need to change for rc.1